docs/modules/ROOT/pages/how-to-guides/proc_maintaining_operator_graphs.adoc

```diff
@@ -0,0 +1,222 @@
+:_content-type: PROCEDURE
+:myfunctionone: maintaining_operator_graphs
+
+[id="maintaining_operator_graphs_{context}"]
+= Maintaining operator graphs using the information on the File-based catalogs
+File-based catalogs (FBC) provide a streamlined approach to store and distribute metadata about operators written for the Operator Lifecycle Manager (OLM). You can use {ProductName} to build, test, and release multiple components from a single FBC repo. 
+
+FBCs represent a departure from the previous method of defining operator graphs using a CSV file. Previously, operator developers defined operator graphs implicitly in CSV files. This made made it difficult to maintain and update the graphs, as any changes required updating the CSV files and restarting the OLM controller.
+
+FBCs simplyfy graph management, offering enhanced control and ownership of operator graphs. With a choice of JSON or YAML formats, File-based catalogs ensure backward compatibility while optimizing efficiency.
+
+The advantages it offers are:
+
+* *Enhance control and simplify graph updates:* With FBC, graph updates are consolidated into a single catalog file. This eliminates the need for iterative updates and simplifies the management of complex graphs. Users can easily manipulate and control their graphs by working with a centralized catalog file.
+
+* *Convenient management and flexibility:* Compared to SQLite, FBC offers a more convenient solution. Rather than adding one set of graph updates at a time, users can define all updates in one place using File-based catalogs. This centralized management capability simplifies the process and improves overall efficiency.
+
+* *Improved ownership and analysis:* FBC provides users with improved ownership and control over their graphs. They offer a clean and declarative template that facilitates comprehensive analysis and sanity checks. Users can easily determine upgrade possibilities and recover from error states, no longer relying on limited configuration options found in localized CSV files.
```

Comment:
I thought the style guide recommended surface level checks instead of sanity? Should we say something different there?